tests/phpunit/includes/TestCase.php

```diff
@@ -248,6 +248,12 @@ protected function assertPostMetaExists( $post_id, $meta_key ) {
 		);
 	}
 
+	protected function assertPostMetaHasValue( $post_id, $meta_key, $meta_value ) {
+		$meta = $this->queryPostMeta( $post_id, $meta_key );
+		$this->assertNotNull( $meta );
+		$this->assertEquals( $meta_value, $meta['meta_value'], "Failted to assert that post $post_id has \"$meta_key\" meta with \"$meta_value\" value." );
```

Comment:
```suggestion
		$this->assertEquals( $meta_value, $meta['meta_value'], "Failed to assert that post $post_id has \"$meta_key\" meta with \"$meta_value\" value." );
```

Reply:
Seems like a misspelling here, but then it _did_ pass. What's causing this to emit "failted"? I couldn't find it in our code, but is it worth noting why that… typo appears? 🤷🏻‍♂️

Reply:
@tofumatt the last parameter is just a friendly message that will be displayed if the first two parameters don't equal.